HLTriggerOffline/Muon/python/hltMuonValidator_cfi.py

```diff
@@ -44,3 +44,12 @@
     genMuonCut  = cms.string("abs(pdgId) == 13 && status == 1"),
     recMuonCut  = cms.string("isGlobalMuon"),
 )
+
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+phase2_common.toModify(hltMuonValidator,
+                       hltPathsToCheck = cms.vstring(
+                           "HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_FromL1TkMuon(_v[0-9]*)?$",
```

Comment:
Same comment here.

Reply:
Hi @srimanob , the correct strings corresponding to the path names can be found here:
https://github.com/cms-sw/cmssw/blob/master/HLTrigger/Configuration/python/HLT_75e33_cff.py#L235-L284

Reply:
Hi @SohamBhattacharya 
I don't expect a full names, but some string format to pick the correct path. This will avoid adding new names anytime you have new paths.